src/main/features/offer/form/models/offer.ts

```diff
@@ -0,0 +1,47 @@
+import { MaxLength, IsDefined, ValidateNested } from 'class-validator'
+import { Serializable } from 'models/serializable'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+import { IsFutureDate } from 'forms/validation/validators/dateFutureConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsValidLocalDate } from 'forms/validation/validators/isValidLocalDate'
+
+export class ValidationErrors {
+  static readonly DATE_REQUIRED: string = 'Please enter a valid date'
+  static readonly DATE_NOT_VALID: string = 'Please enter a valid date'
+  static readonly FUTURE_DATE: string = 'Enter an offer date in the future'
+  static readonly OFFER_REQUIRED: string = "You haven't made your offer"
+  static readonly OFFER_TEXT_TOO_LONG: string = "You've entered too many characters"
```

Comment:
> 'You’ve entered too many characters'

Reply:
this is what @BenClancy confirmed

Reply:
you have the quote mark not an apostrophe, same with the above line